project/SbtUpdatesSettings.scala

```diff
@@ -6,7 +6,8 @@ import sbt.Keys.*
 object SbtUpdatesSettings {
 
   lazy val sbtUpdatesSettings = Seq(
-    dependencyUpdatesFailBuild := true,
+    dependencyUpdatesFailBuild := false,
```

Comment:
this may have been me - we can stick this back to `true` now I reckon

Reply:
this was actually me, my bad